scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/TerminationData.java

```diff
@@ -136,6 +136,12 @@ void handleTermination(final SchedulingService service) throws IOException, Clas
                 logger
                         .info("Cannot terminate task launcher for task '" + taskData.getTask().getId() + "'",
                                 t);
+                try {
+                    logger.info("Task launcher that cannot be terminated is identified by " +
+                            taskData.getLauncher().toString());
+                } catch (Throwable ignore) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1181)